### PR TITLE
Iigeregi dns onprem - two regions

### PR DESCRIPTION
**Pull request by iigeregi**

### CODE REVIEW

**Comment by juliocc** [comment]
Nice! can you make sure all tests pass before we merge? Ping me if you need help running the tests

@averbuks @ludoo can you take a look?

File: networking/onprem-google-access-dns/main.tf copy [hunk 1]
```diff
@@ -0,0 +1,332 @@
+/**
```
Comment (ludoo) [inline networking/onprem-google-access-dns/main.tf copy:1]: can you remove this?

File: networking/onprem-google-access-dns/variables.tf [hunk 2]
```diff
@@ -18,16 +18,19 @@ variable "bgp_asn" {
   description = "BGP ASNs."
```
Comment (ludoo) [inline networking/onprem-google-access-dns/variables.tf:18]: I would redesign variables, using a single "regions" variable where key is the region name, values an object with parameters for each region (ASN, IP range, etc.).

Then parameterize main.tf so we create resources for 1-n regions.

**Review by ludoo — COMMENTED** [review]
This is super nice. As I mentioned in a comment on the variables, I would parameterize this to accept 1-n regions.

And as Julio said, please fix tests. :) Ping us if you need help with those.

**Comment by ludoo** [comment]
I'm closing this since Aleks sent a new revised PR, let me know if it needs to be reopened.

**Pull request closed without merging**